krun/util.py

```diff
@@ -180,3 +181,15 @@ def print_session_info(config):
             print("  %s" % k)
     else:
         print("  No keys skipped")
+
+
+def make_heat():
+    """A loop which spins, attempting to make heat.
+
+    Used when the CPU is too cool to run a benchmark."""
+
+    until = time.clock() + 1  # in roughly one second
+    while True:
+        # Waste cycles
+        if time.clock() > until:
```

Comment:
Well, in this case, we can make it still do *something* sensible on PyPy. I'm not super keen on forcing one implementation over the other unless there's a compelling reason.